src/ui/components/Card/Card.scss

```diff
@@ -45,9 +46,7 @@
   a:visited {
     color: $link-color;
     display: block;
-    font-variant: all-small-caps;
     font-weight: normal;
-    padding: 10px;
```

Comment:
Hmm, this will reduce the tap target on the link itself. Before it would take up the entire footer which was a lot nicer; now if you tap on the edge of the footer it won't register a click/tap event. I think we might need to change the `<Card>` component to accept a `footerUrl` and wrap the contents in a link if we want the link to fill the entire footer.

Unless React can tell what the element inside `.Card-footer` is and apply a style that either pads the link or the `.Card-footer` class itself based on the element (though that is certainly more magically and less obvious).

So I think we need to change `<Card footer={footerHTMLWithALink}>` to `<Card footer={footerHTML} footerLink={'/someplace'}>`.

We can do that outside this PR if you want as it's not related, but we'll have to file an issue first.